--- jigdo-0.7.3.orig/scripts/jigdo-lite
+++ jigdo-0.7.3/scripts/jigdo-lite
@@ -46,10 +46,14 @@
 #______________________________________________________________________
 
 # isURI <string>
-# Returns 0 (true) if the supplied string is a HTTP/FTP URL, otherwise 1
+# Returns 0 (true) if the supplied string is a HTTP/HTTPS/FTP/FILE
+# URL, otherwise 1
 isURI() {
   case "$1" in
-    http:*|ftp:*|HTTP:*|FTP:*|file:*|FILE:*) return 0;;
+    [hH][tT][tT][pP]:*) return 0;;
+    [hH][tT][tT][pP][sS]:*) return 0;;
+    [fF][tT][pP]:*) return 0;;
+    [fF][iI][lL][eE]:*) return 0;;
     *) return 1;
   esac
 }
@@ -596,7 +596,7 @@ imageDownload() {
     for pass in x xx xxx xxxx xxxxx xxxxxx xxxxxxx xxxxxxxx; do
       $jigdoFile print-missing-all --image="$image" --jigdo="$jigdoF" \
         --template="$template" $jigdoOpts $uriOpts \
-      | egrep -i '^(http:|ftp:|$)' >"$list"
+      | egrep -i '^(http:|https:|ftp:|$)' >"$list"
       missingCount=`egrep '^$' <"$list" | wc -l | sed -e 's/ *//g'`
       # Accumulate URLs in $@, pass them to fetchAndMerge in batches
       shift "$#" # Solaris /bin/sh doesn't understand "set --"
